### DISCUSSION TITLE
Changing the ping method for mac computers

### DISCUSSION THREAD

**Discussion by MDo3** [AlexGustafsson/homebridge-wol#204] [General]
Is there any way to ping the mac system other than the current one? 

`"pingCommand": "ssh macbook.local 'if [[ $(pmset -g powerstate IODisplayWrangler | tail -1 | cut -c29) -lt 4 ]]; then; exit 1; else; echo 1; fi;'"`

The above script is based on the screen work, which is not a good solution. Personally, he often uses the computer as a server, leaving it with the monitor turned off, which makes the HomeKit switch status show a false status.

**Comment by AlexGustafsson** [comment]
I don't have any alternative ready to go, but you can always try to identify if there is some sort of program running, if a user is connected, if the computer's uptime is more than some variable etc. Feel free to leave any findings here. It's always helpful to others.
